### DISCUSSION TITLE
http method装饰器的一个细节建议

### DISCUSSION THREAD

**Discussion by zanminkian** [midwayjs/midway#3531] [General / 闲聊]
![image](https://github.com/midwayjs/midway/assets/35495628/fbe32f9d-b0ac-45df-b0ea-b01dc010b9d7)
根据 mizilla 的文档 https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods。 我们并不存在名为 Del 的 http method。在一众没有缩写的 http 方法里面出现一个缩写的 `@Del` 显得非常诡异。

**Comment by czy88840616** [comment]
最开始 v2 版本装饰器因为是小写，碰到了 delete 是保留关键字，为了避免歧义，使用了 @del，v3 因为延续了，就只是换成了大写。

**Reply by zanminkian** [reply]
有道理。但还是建议给Del加个deprecated，然后新增Delete装饰器。
